operator/v1/0000_20_kube-apiserver-operator_01_config.crd.yaml-patch

```diff
@@ -0,0 +1,3 @@
+- op: replace
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/managementState/pattern
+  value: "^(Managed|Force)$"
```

Comment:
could you add a note in the readme CRD section on how to use these (or just link to the project)

Reply:
Good idea.

Reply:
done. Ptal.